File: PythonPrograming/selectionSort.py
def selectionSort(l):
    for i in range(len(l)):
        minpos=i
        for j in range(i,len(l)):
            if(l[minpos]>l[j]):
                minpos=j
        (l[i],l[minpos])=(l[minpos],l[i])
    return(l)

if __name__=="__main__":
    print(selectionSort([2,4,5,1,3,7,6,9,8]))

File: JavaPrograming/String/SelectionSort.java
import java.util.Arrays;

public class SelectionSort {
    public static void main(String[] args){
        int[] a={34 ,56,12,34,12,7,83};
        System.out.println(selectionSort(a));
    }
    public static String selectionSort(int []a){
        for(int i=0;i<a.length;i++){
            int minpos=i;
            for(int j=i;j<a.length;j++){
                if(a[j]<a[minpos]){
                    minpos=j;
                }
            }
            int temp=a[minpos];
            a[minpos]=a[i];
            a[i]=temp;
        }
        return(Arrays.toString(a));

    }
}
